test/apis/v1alpha1/suite_test.go

```diff
@@ -23,14 +23,16 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
 
-	v1alpha1 "go.goms.io/fleet-networking/api/v1alpha1"
+	fleetnetv1alpha1 "go.goms.io/fleet-networking/api/v1alpha1"
 )
 
 var (
-	hubTestEnv *envtest.Environment
-	hubClient  client.Client
-	ctx        context.Context
-	cancel     context.CancelFunc
+	hubTestEnv          *envtest.Environment
+	hubClient           client.Client
+	ctx                 context.Context
+	cancel              context.CancelFunc
+	testNamespace       = "testnamespace"
+	nameWithInvalidSize = "abcdef-123456789-123456789-123456789-123456789-123456789-123456789-123456789"
```

Comment:
please declare this varible in api_validation_integration_test.go file where you used it.